lib/blueprinter/view_collection.rb

```diff
@@ -29,8 +29,10 @@ def fields_for(view_name)
       (identifier_fields + sorted_fields).reject { |field| excluded_fields.include?(field.name) }
     end
 
-    def transformers(view_name)
-      views[view_name].transformers
+    def transformers_for(view_name)
```

Comment:
The meaning of `transformers_for` is unclear to me. Shouldn't this remain `transformers`?

Reply:
Retained as it is